### DISCUSSION TITLE
Formação flutter - Migração da versão

### DISCUSSION THREAD

**Discussion by leandesenv** [balta-io/forum#472] [Tira Dúvidas]
Fala pessoal,

Alguém pode me dar um help? Estou fazendo o curso de formação em flutter, tô curtindo pra caramba. 

Gostaria de saber se alguém conseguiu estudar o conteúdo e migrar o código fonte disponível do curso para o flutter 2+ ou até mesmo para o 3. Recebo alguns erros na versão do sdk do android, mas esse é tranquilo de resolver. Caso alguém tenha conseguido, poderiam indicar um material bacana ou mesmo o código migrado? Seria de grande ajuda! Pensei em codar junto com a versão do curso, instalando o sdk apresentado. Pra isso teria que apontar pra essa pasta específica, baixando a versão do flutter do curso (se eu nao me engano 1.3). 
Quem puder ajudar! Forte abraço!

**Comment by Azharhayat271** [comment]
flutter keep changing its versions so many library or SDK become deprecated for the latest version so i would recommend you to take the lastest course that is recorded in the 2023 or later.
